packages/fxa-settings/src/components/Settings/FlowContainer/index.tsx

```diff
@@ -31,21 +31,21 @@ export const FlowContainer = ({
     : ftlMsgResolver.getMsg('flow-container-back', 'Back');
   return (
     <div
-      className="max-w-lg mx-auto mt-6 p-10 tablet:my-10 flex flex-col items-start bg-white shadow tablet:rounded-xl"
+      className="max-w-lg mx-auto mt-6 p-10 mobileLandscape:my-10 flex flex-col items-start bg-white shadow mobileLandscape:rounded-xl"
```

Comment:
Q: Why the change from `tablet:*` to `mobileLandscape:*`?
Or more importantly, do we need to modify the 46 other instances of `tablet:*` [in 28 unique files]?

```sh
git grep -Pn "tablet:(\w|\d|-)+" packages/ | wc -l # 46 matches
git grep -Pl "tablet:(\w|\d|-)+" packages/ | wc -l # 28 files
```

Reply:
`tablet` is a bit of an outlier compared to our other media query configs - it has an upper and lower bound, compared to others like `mobileLandscape` and `desktop` which only have a minimum. In this case we'd want to styling to apply for anything larger than mobile portrait, so `mobileLandscape` is the better choice.

I'm uncertain if other uses of `tablet` intentionally targeted that specific format, or if it was applied with the intention of applying the styling to anything "tablet and larger".